lib/seo_renderer.dart

```diff
@@ -1,5 +1,9 @@
-export 'helpers/scroll_listener/renderer_scroll_listener.dart';
-export 'helpers/utils.dart';
-export 'renderers/image_renderer/image_renderer.dart';
-export 'renderers/link_renderer/link_renderer.dart';
-export 'renderers/text_renderer/text_renderer.dart';
+export 'helpers/robot_detector_vm.dart'
```

Comment:
I think we can Import `web` files as default and vm if `io` is present
Somewhat like this ?
```
export 'helpers/robot_detector_web.dart'
    if (dart.library.io) 'helpers/robot_detector_vm.dart';
export 'renderers/image_renderer/image_renderer_web.dart'
    if (dart.library.io) 'renderers/image_renderer/image_renderer_vm.dart';
export 'renderers/link_renderer/link_renderer_web.dart'
    if (dart.library.io) 'renderers/link_renderer/link_renderer_vm.dart';
export 'renderers/text_renderer/text_renderer_web.dart'
    if (dart.library.io) 'renderers/text_renderer/text_renderer_vm.dart';
export 'renderers/text_renderer/text_renderer_style.dart';
```
Is it because if someone is backtracking the source code through constructors/anything it will lead them to vm file, as we are importing vm files unconditionally. That would make bit difficult for them to reach to the source. 
Ideally `web` files should be available through backtracking. If you think current approach is good for some other reason, let me know :)

Reply:
You're right 👍  I didn't have any reason for this.